pkg/scheduler/framework/plugins/volumebinding/binder.go

```diff
@@ -247,10 +259,68 @@ func NewVolumeBinder(
 	return b
 }
 
+// GetEligibleNodes returns a list of eligible nodes by name based on the bound claims of the pod to reduce down
+// the list of nodes in subsequent scheduling stages
+func (b *volumeBinder) GetEligibleNodes(boundClaims []*v1.PersistentVolumeClaim) (sets.String, error) {
+	eligibleNodes := sets.NewString()
+	lock := sync.RWMutex{}
+	if len(boundClaims) == 0 {
+		return eligibleNodes, nil
+	}
+
+	nodes, err := b.nodeLister.List(labels.Everything())
```

Comment:
is it possible to translate PV's nodeAffinitySelector to a `labels.Selector`, and then pass-in here?